### DISCUSSION TITLE
How to make "docker.exe" in "rancher desktop" use "daemon.json"

### DISCUSSION THREAD

**Discussion by jack-liew** [rancher-sandbox/rancher-desktop#3557] [Q&A]
### Actual Behavior

HI:
When running docker build command in PowerShell and getting:
```
Nuget connection attempt failed:
"Unable to load the service index for source:https://api.nuget.org/v3/index.json"
```
I get a solution:
edit "c:\ProgramData\Docker\config\daemon.json" and looks like this:
```
{
  "registry-mirrors": [],
  "insecure-registries": [],
  "debug": true,
  "experimental": true,
  "dns": ["8.8.8.8"],
}
```

But, how should i do in rancher desktop? I can not found any "daemon.json".

Thanks

### Steps to Reproduce

dockerfile: 
run dotnet  restore

### Result

 I can not found ""c:\ProgramData\Docker\config\daemon.json" and any "daemon.json"

### Expected Behavior

hop  fix  it .

### Rancher Desktop Version

1.62

### Rancher Desktop K8s Version

1.25.4

### Which container engine are you using?

moby (docker cli)

### What operating system are you using?

Windows

### Operating System / Build Version

windows 11

### What CPU architecture are you using?

x64

### Linux only: what package format did you use to install Rancher Desktop?

None

### Windows User Only

no

**Comment by ericpromislow** [comment]
This looks more like a docker issue than a Rancher Desktop issue -- the only instance of `nuget` in the current code base is in a comment dealing with a Windows installer, and has nothing to do with managing containers.

@gunamata - Could you rule out that this isn't an issue involving the VS Code plugin?

**Comment by mook-as** [comment]
Hi! Rancher Desktop does not have a `C:\ProgramData\Docker\config\daemon.json` (because we're not Docker). You'll need to edit `/etc/daemon.json` (by using `rdctl shell` first, then something like `vi /etc/daemon.json`) instead.

**Reply by jandubois** [reply]
It should be `/etc/docker/daemon.json`. And you need to use `sudo` to edit it.

**Reply by jack-liew** [reply]
> Hi! Rancher Desktop does not have a `C:\ProgramData\Docker\config\daemon.json` (because we're not Docker). You'll need to edit `/etc/daemon.json` (by using `rdctl shell` first, then something like `vi /etc/daemon.json`) instead.

Thank you, but It does not work.

After creating this file: "/etc/docker/daemon.json", an error is reported:
![image](https://user-images.githubusercontent.com/102201538/207601338-9bf77df7-dacd-4db1-a192-63c8c25ff31c.png)
